package.json

```diff
@@ -313,9 +312,11 @@
     "eslint-config-amo": "^1.8.3",
     "eslint-plugin-amo": "^1.9.1",
     "eslint-plugin-promise": "^4.0.0",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
     "file-loader": "^2.0.0",
     "flow-bin": "^0.86.0",
     "glob": "^7.1.1",
+    "html-webpack-plugin": "^3.2.0",
```

Comment:
This is not used but it is a peer dependency of `webpack-subresource-integrity`.